modules/aruco/src/aruco.cpp

```diff
@@ -1330,6 +1329,33 @@ void GridBoard::draw(Size outSize, OutputArray _img, int marginSize, int borderB
 
 
 /**
+*/
+Ptr<Board> Board::create(InputArrayOfArrays objPoints, Ptr<Dictionary> &dictionary, InputArray ids) {
+
+    Mat ids_mat = ids.getMat();
+    std::vector<int> ids_vector;
+    for (int i = 0; i < ids_mat.rows; i++) {
+        ids_vector.push_back(ids_mat.at<int>(i));
+    }
+
+    std::vector< std::vector< Point3f > > obj_points_vector;
+    for (unsigned int i = 0; i < objPoints.total(); i++) {
+        std::vector<Point3f> corners;
+        Mat corners_mat = objPoints.getMat(i);
+        for (int j = 0; j < 4; j++) {
+            corners.push_back(corners_mat.at<Point3f>(j));
```

Comment:
You could safe the temporary `corners_mat` by using `objPoints.at<Point3f>(i, j)`.